src/quote/composer.json

```diff
@@ -6,20 +6,21 @@
         "php": ">= 8.3",
```

Comment:
I think this file doesn't belong to this PR

Reply:
I reset this file to what is on main.